news/3728.feature

```diff
@@ -0,0 +1,7 @@
+Add support for variable expansion in URLs in the requirement file. This allows
```

Comment:
Same note as in the documentation about the example -- the following would be enough here:

```
pip now supports environment variable expansion in requirement files using ``${VARIABLE}`` syntax.
```

Reply:
Also, my reading of the code suggests the expansion is not only supported in URLs but really anywhere.